src/components/App.js

```diff
@@ -14,7 +23,21 @@ export default class App extends Component {
   }
 
   render() {
-    return <div>Hello App</div>
+    return (
+      <Wrapper>
+        <Header text="STEMfluence" />
+        <Card
+          imgURL={'https://source.unsplash.com/random'}
```

Comment:
why is this in curly brackets? See also the rest of the props. If this is the place you're defining them, you only need curly brackets for booleans.
Formatted like this, it looks as if you're already importing this from this.state or this.props.